website/assets/styles/pages/articles/basic-webinar.less

```
@@ -0,0 +1,753 @@
+    width: 100%;
+  }
+  .markdown-heading:hover {
+    .markdown-link {
+      height: 16px;
+      vertical-align: middle;
+      margin-left: 8px;
+      content: url('/images/icon-link-16x16@2x.png');
+    }
+  }
+
+  @keyframes copiedText {
```

Stylelint (17.7.0)

[error] Expected keyframe name "copiedText" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)